pull_seqr_pedigrees.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import getpass
+import argparse
+
+
+def pull_project_peds(args):
+    """
+    Uses the seqr API to retrieve a list of projects' pedigrees. Writes a single pedigree,
+    'pulled_seqr_pedigrees.txt', using the retrieved pedigrees. Failing projects are written
+    to 'projects_not_pulled.txt'
+    """
+    email=args.email
+    projects=args.projects
+
+    with requests.Session() as s:
+        p = s.post(
+            "https://seqr.broadinstitute.org/api/login", 
+            json={
+                'email': email,
+                'password': getpass.getpass(),
+            }
+        )
+        final_ped = open('seqr_pedigrees.txt', 'w')
+        final_ped.write('Project_GUID\tFamily_ID\tIndividual_ID\tPaternal_ID\tMaternal_ID\tSex\n') 
+        errors = open('projects_not_pulled.txt', 'w')
+
+        with open(projects, 'r') as project_list:
+            for project_guid in project_list:
+                project_guid = project_guid.rstrip()
+                r = s.get(f'https://seqr.broadinstitute.org/api/project/{project_guid}/details') 
+                if r.status_code != requests.codes['ok']:
+                    errors.write(f'{project_guid}\n')
+                else:
+                    fams = r.json()['familiesByGuid']
+                    fams_dict = {}
+                    for key in fams:
```

Comment:
both here and in the loop below, since key isn't used except to do fams[key], would instead do 
for fam in fams.values()

Reply:
Much better!